src/azure-cli/azure/cli/command_modules/synapse/_params.py

```diff
@@ -56,18 +56,24 @@ def load_arguments(self, _):
             c.argument('sql_admin_login_password', options_list=['--sql-admin-login-password', '-p'],
                        help='The sql administrator login password.')
             c.argument('tags', arg_type=tags_type)
+            c.argument('key_name', help='The workspace key name.')
```

Comment:
>The workspace key name. [](start = 41, length = 23)

Shall we give more concrete description to make it clear that it is the workspace display key name instead of the name of key vault?